src/gt4py/next/program_processors/runners/dace_fieldview/gtir_to_sdfg.py

```diff
@@ -184,7 +184,7 @@ def _add_storage(
 
         # TODO: unclear why mypy complains about incompatible types
         assert isinstance(symbol_type, (ts.FieldType, ts.ScalarType))
-        self.symbol_types[name] = symbol_type
+        self.global_symbols[name] = symbol_type
```

Comment:
Just out of curiosity, why did you rename the `symbol_table` which maps symbols to their type I guess, to `global_symbols` I would not expect that this gives me their type?